src/Resources/config/blast.yml

```diff
@@ -1612,42 +1612,154 @@ parameters:
                 pattern: librinfo/ecommerce/channel
             # SHOW
             Sonata\AdminBundle\Show\ShowMapper:
+                remove:
+                    id: ~
+                    color: ~
+                    baseCurrency: ~
+                    defaultLocale: ~
+                    defaultTaxZone: ~
+                    taxCalculationStrategy: ~
+                    currencies: ~
+                    locales: ~
+                    themeName: ~
+                    contactEmail: ~
+                    skippingShippingStepAllowed: ~
+                    skippingPaymentStepAllowed: ~
+                    accountVerificationRequired: ~
+                    updatedAt: ~
+                    code: ~
+                    name: ~
+                    description: ~
+                    enabled: ~
+                    hostname: ~
+                    createdAt: ~
+                    updatedAt: ~
                 add:
                     main_tab:
                         main_group:
                             code: ~
                             name: ~
-                            color: ~
                             description: ~
                             enabled: ~
                             hostname: ~
                             createdAt: ~
                             updatedAt: ~
-                            #id: ~
-
             # FORM
             Sonata\AdminBundle\Form\FormMapper:
                 remove:
                     id: ~
-                    name: ~
                     code: ~
+                    createdAt: ~
                 add:
-                    main_tab:
-                        main_group:
-                            code: ~
-                            name: ~
-                            color: ~
+                    form_tab_general:
+                        '':
```

Comment:
Could you put « form_group_general » instead of « '' » ?

Reply:
Instead of form_group_general, please find another name, starting with « form_group_ » and add the corresponding translation